ContractsApi.Tests/V1/Controllers/ContractsApiControllerTests.cs

```diff
@@ -210,5 +222,32 @@ public async Task PatchContractReturns404NotFoundIfNoContractIsFound()
 
             response.Should().BeOfType(typeof(NotFoundResult));
         }
+
+        [Fact]
+        public async Task PatchContractReturnsErrorIfValidationFails()
+        {
+            var mockGuid = new Guid();
+            var mockRequestObject = _fixture.Create<EditContractRequest>();
+
+            var today = DateTime.Today;
+            var tomorrow = today.AddDays(1);
+
+            mockRequestObject.StartDate = tomorrow;
+            mockRequestObject.HandbackDate = today;
+
+            _mockPatchContractUseCase
+                .Setup(x => x.ExecuteAsync(mockGuid, mockRequestObject, It.IsAny<string>(), It.IsAny<Token>(),
+                    It.IsAny<int?>())).ReturnsAsync((ContractResponseObject) null);
+
+            var response = await _classUnderTest.PatchContract(mockGuid, mockRequestObject).ConfigureAwait(false);
+            var statusCode = GetStatusCode(response);
+            statusCode.Should().Be(400);
```

Comment:
Would be good to also add to the assertion that the correct error message is sent

Reply:
Addressed in [ad05ee1](https://github.com/LBHackney-IT/contracts-api/pull/48/commits/ad05ee1d6c4c9ced5e1af30a9601fe03bc652ce8).